_layouts/blog-home.html

```diff
@@ -0,0 +1,64 @@
+<!doctype html>
+
+{% include head.html %}
+
+<body>
+    {% include header-extended.html %}
+    <section class="usa-section bg-blue-80v">
+        <div class="grid-container">
+            <h1 class="font-heading-3xl text-base-lightest">{{ page.title }}</h1>
+            <p class="font-alt-xl text-base-lightest">{{ page.description }}</p>
+        </div>
+    </section>
+
+    <main id="main-content">
+        <div class="grid-container padding-y-6">
+            <div class="grid-row grid-gap">
+                <div class="tablet:grid-col-4 tablet:order-2">
+                    {% include search-small.html %}
+                    <h2 class="margin-top-3">Tags</h2>
+                    <nav aria-label="Secondary navigation">
+                        <ul class="usa-sidenav margin-bottom-3">
+                            {% for item in page.blog-categories %}
+                            <li class="usa-sidenav__item">
+                                <a href="javascript:void(0);">{{ item.name }}</a>
+                            </li>
+                            {% endfor %}
+                        </ul>
+                    </nav>
+                </div>
+                <div class="tablet:grid-col-8 tablet:order-1">
+                    <ul class="usa-card-group">
+                        {% for item in page.blog-posts %}
+                        <li class="usa-card">
+                            <div class="usa-card__container">
+                                <header class="usa-card__header">
+                                    <h2 class="usa-card__heading">{{ item.title }}</h2>
```

Comment:
Sorry! I realized another thing looking at it again. Can you add a dummy link to the card title?

Reply:
@pglevy no problem! Changes have been made